tools/linker/MobileExtensions.cs

```diff
@@ -89,7 +89,7 @@ static bool HasGeneratedCodeAttribute (ICustomAttributeProvider provider, Derive
 					return rv;
 			}
 
-			return GetBindingImplAttribute (provider, context?.GetCustomAttributes (provider, Namespaces.ObjCRuntime, "BindingImplOptions"));
+			return GetBindingImplAttribute (provider, context?.GetCustomAttributes (provider, Namespaces.ObjCRuntime, "BindingImplAttribute"));
```

Comment:
that must have cause other issues, right ?

Reply:
Should have, but as far as I can tell, didn't 😒